relational_database/homework.py

```diff
@@ -150,7 +168,10 @@ def task_12_list_suppliers_from_specified_countries(cur):
 
     Returns: 8 records
     """
-    pass
+    cur.execute("""SELECT SupplierID, SupplierName, ContactName, City, Country 
+                 FROM Suppliers 
```

Comment:
Find one way to format strings, because here you format it so, below different...